### DISCUSSION TITLE
error during syncRequiredMachineConfigPools during update from 4.11 to 4.12

### DISCUSSION THREAD

**Discussion by dkoci** [okd-project/okd#1504] [General]
Hi.
I have some issue with update from OKD 4.11 to 4.12.
`Unable to apply 4.12.0-0.okd-2023-01-21-055900: error during syncRequiredMachineConfigPools: [timed out waiting for the condition, error pool master is not ready, retrying. Status: (pool degraded: true total: 3, ready 0, updated: 0, unavailable: 3)]`

Can someone help me with that? Must-gather logs are available at https://we.tl/t-GSvHXMS1E8

Thank you

**Comment by vrutkovs** [comment]
```
2023-02-10T20:42:19.775013763+01:00 E0210 19:42:19.774959    2521 writer.go:200] Marking Degraded due to: failed to update OS to quay.io/openshift/okd-content@sha256:01d90a996a2e78a0486616a0adba0733db428f5a1074976054cedff62f17b2ac : error running rpm-ostree rebase --experimental ostree-unverified-registry:quay.io/openshift/okd-content@sha256:01d90a996a2e78a0486616a0adba0733db428f5a1074976054cedff62f17b2ac: [0m[31merror: [0mNo enabled repositories
```
See #1483

**Comment by dkoci** [comment]
`rpm-ostree uninstall glusterfs glusterfs-fuse` fixed worker nodes. But there are still problems with master mcp:

`Failed to resync 4.12.0-0.okd-2023-01-21-055900 because: error during syncRequiredMachineConfigPools: [timed out waiting for the condition, error pool master is not ready, retrying. Status: (pool degraded: true total: 3, ready 0, updated: 0, unavailable: 3)]`

oc describe mcp master:
` Message:               Node okd-master01.ts.telekom.si is reporting: "error running rpm-ostree kargs --append=mitigations=off --append=intel_pstate=disable: \x1b[0m\x1b[31merror: \x1b[0mNo packages in transaction\n: exit status 1", Node okd-master02.ts.telekom.si is reporting: "error running rpm-ostree kargs --append=mitigations=off --append=intel_pstate=disable: \x1b[0m\x1b[31merror: \x1b[0mNo packages in transaction\n: exit status 1"`

**Comment by dkoci** [comment]
I tried to remove machine-config related to `migrations=off` and `intel_pstate=disable` kargs but machine config pool recreates it every time.
